crf/src/main/java/org/sagebase/crf/step/CrfHeartRateStepLayout.java

```diff
@@ -496,31 +508,26 @@ protected void showCompleteUi() {
         crfMessageTextView.setVisibility(View.INVISIBLE);
 
 
+        int hrToDisplay = 0;
         if (!bpmList.isEmpty()) {
-            int bpmSum = 0;
-            for (BpmHolder bpmHolder : bpmList) {
-                bpmSum += bpmHolder.bpm;
+            if (step.isHrRecoveryStep) {
```

Comment:
So... for recovery, I am showing the "last" heart rate on iOS. Basically, it seemed confusing to jump to showing the peak (that was originally on the next screen but that screen was removed).